gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,7 @@
+# checksum was taken from https://gradle.org/release-checksums
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-all.zip
-distributionSha256Sum=97a52d145762adc241bad7fd18289bf7f6801e08ece6badf80402fe2b9f250b1
+distributionSha256Sum=db9c8211ed63f61f60292c69e80d89196f9eb36665e369e7f00ac4cc841c2219
```

Comment:
I guess this reorder is intentional to keep properties in sorted order.

Reply:
yes